src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -281,21 +281,24 @@ namespace OpenMS
         AASequence seq = h.getSequence();
         double precursor_neutral_mass = seq.getMonoWeight();
 
-        Int scan_index;
+        int scan_index = count;
+        int scan_nr = 0;
+        
         if (lookup.empty())
         {
           if (it->metaValueExists("RT_index")) // Setting metaValue "RT_index" in XTandemXMLFile in the case of X! Tandem.
           {
-            scan_index = it->getMetaValue("RT_index");
-          }
-          else
-          {
-            scan_index = count;
+            scan_index = it->getMetaValue("RT_index");            
           }
+          scan_nr = scan_index + 1;          
```

Comment:
can you explain why you add `+1` here, either the `scan_index` was just derived from the `RT_index` or it was set to `count` before which starts already with 1